### PR TITLE
release 0.61.0

### PR DESCRIPTION
**Pull request by RenkuBot**
Release 0.61.0

This PR is auto-generated by [actions/github-script](https://github.com/actions/github-script).

/deploy

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,5 +1,74 @@
...
+- **UI**: Enable the use of custom images that don’t contain Jupyter, streamlining the image-building process and allowing for the use of “off-the-shelf” images (`#3341 <https://github.com/SwissDataScienceCenter/renku-ui/pull/3341>`__).
+- **Sessions**: Enable running session images that do not contain Jupyter in them.
+- **Amalthea**: Brand new operator for sessions with images that do not have to contain Jupyter
+
+Internal Changes
+~~~~~~~~~~~~~~~~
+
+**Improvements**
+
+- **Data services**: Add support for OAuth storage providers
```
Comment (rokroskar) [inline CHANGELOG.rst:46]: Wait what?? What does this mean??

Comment (leafty) [reply CHANGELOG.rst:46]: > This PR adds three new OAuth2 providers, Google Drive, Microsoft OneDrive and Dropbox.
I tested these with some hacked together code in this PR and after confirming they worked, undid the hacky changes. As such, this PR doesn't do anything. Or rather, it'd allow admins to set up OAuth2 apps and for users to connect to them, but once connected, the users can't use them anywhere.
There will be a follow-up build including UI changes that uses these changes to automatically generate tokens for cloud storage for users.
https://github.com/SwissDataScienceCenter/renku-data-services/pull/383

**Review by rokroskar — CHANGES_REQUESTED** [review]

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,5 +1,74 @@
...
+- **Data services**: Move notebooks code to data services
+- **Helm chart**: Consolidate and revamp network policies
+
+**Bug Fixes**
+
+- **Gateway**: Pass on session cookie to data services for anonymous session authentication
+- **Data services**: Correct pagination for namespaces
+- **Data services**: Add creation date and created_by for namespaces
```
[the remark this answers is not in the document]
Comment (Panaetius) [reply CHANGELOG.rst:54]: search does not have/get this information (yet)

[the remark this answers is not in the document]
Comment (rokroskar) [reply CHANGELOG.rst:54]: well lets make that happen! :)

**Review by rokroskar — APPROVED** [review]

**Review by aledegano — APPROVED** [review]

**Review by lokijuhy — APPROVED** [review]

**Pull request merged**